internal/driver/common/driver.go

```diff
@@ -5,6 +5,14 @@ import (
 	"fyne.io/fyne/v2/internal/cache"
 )
 
+const (
+	// moved here from the platform driver packages to allow
+	// referencing in widget package without an import cycle
+
+	DesktopDoubleClickDelay = 300 // ms
```

Comment:
This probably could be exposed just as DoubleClickDelay with build constraints setting it appropriately.
Just a thought, it would take the logic out of the Entry to switch based on platform

Reply:
How about making it a driver public API? `driver.DoubleClickDelay() time.Duration` So if other 3rd party widgets needed to have triple click - or to emulate DoubleTapped with Tapped + time comparisons (as I need to do in Supersonic to have list rows instantly tappable while also being DoubleTappable), then they could be consistent with the builtin ones

Reply:
Of course in the meantime until 2.5 we still need a solution for Entry if we want to get this in a 2.4.x point release. But if we have a long-term plan we could just duplicate the constant 🤷‍♂️

Reply:
It's in the 2.5 milestone not 2.4.x currently. We don't usually add features in point release and I'm not sure we could list this as a bug fix ;)

Reply:
Since this is targeted for 2.5 anyway, I reworked with a new public API DoubleTapDelay on the driver interface. This also allows for 3rd party widgets to be able to match double click delay for whatever custom logic they may have. And is also future compatible for getting the value from the OS instead of having it be a compile-time constant, for OSes that provide an API.